src/ensembl/src/content/app/browser/browser-region-editor/BrowserRegionEditor.test.tsx

```diff
@@ -124,11 +131,11 @@ describe('<BrowserRegionEditor />', () => {
 
       const locationStartInput = getCommaSeparatedNumber(faker.random.number());
       const locationEndInput = getCommaSeparatedNumber(faker.random.number());
-      const startError = faker.lorem.words();
-      const endError = faker.lorem.words();
+      const startError = 'start error message';
+      const endError = 'end error message';
```

Comment:
Just curious to know why we aren't using random words anymore for strings in tests?

Reply:
@imransl thanks for noticing 😄 

Random values turn out to be pretty difficult to debug when a test breaks. When I was migrating the tests and was checking that the rendered output was correct or that callbacks were called with correct arguments, seeing random Latin words in the error messages of failed tests (and some tests did fail for me at first) wasn't really helpful for discovering where the problem lies. In this component, for example, it's far easier to understand which wrong error message has been shown if they are explicitly written down and not generated randomly. 

I know that some people prefer not to use random values at all, choosing 100% reproducibility of a test over the fact that the test satisfies all imaginable inputs. I don't have sufficient confidence to advise whether we should stop using random values. At this point, I think, it's more of a gut feeling.

Reply:
@azangru Thanks for the explanation. If it helps us debug test more easily, then this is definitely better 😄